core/src/main/java/org/opensearch/sql/expression/datetime/DateTimeFunction.java

```diff
@@ -1063,6 +1094,38 @@ private ExprValue exprTimeToSec(ExprValue time) {
     return new ExprLongValue(time.timeValue().toSecondOfDay());
   }
 
+  /**
+   * UTC_DATE implementation for ExprValue.
+   *
+   * @param functionProperties FunctionProperties.
+   * @return ExprValue.
+   */
+  private ExprValue exprUtcDate(FunctionProperties functionProperties) {
+    return new ExprDateValue(exprUtcTimeStamp(functionProperties).dateValue());
+  }
+
+  /**
+   * UTC_TIME implementation for ExprValue.
+   *
+   * @param functionProperties FunctionProperties.
+   * @return ExprValue.
+   */
+  private ExprValue exprUtcTime(FunctionProperties functionProperties) {
+    return new ExprTimeValue(exprUtcTimeStamp(functionProperties).timeValue());
+  }
+
+  /**
+   * UTC_TIMESTAMP implementation for ExprValue.
+   *
+   * @param functionProperties FunctionProperties.
+   * @return ExprValue.
+   */
+  private ExprValue exprUtcTimeStamp(FunctionProperties functionProperties) {
+    var zdt = ZonedDateTime.now(functionProperties.getQueryStartClock())
+        .withZoneSameInstant(ZoneId.of("UTC"));
```

Comment:
Try using `ExprTimestampValue.ZONE`

Reply:
> Try using `ExprTimestampValue.ZONE`

This is supposed to be UTC all the time, isn't it?

Reply:
Well, what was the purpose of timezone in `ExprTimestampValue` class? Was it planned as a non-static?

Reply:
> Well, what was the purpose of timezone in `ExprTimestampValue` class? Was it planned as a non-static?

Not pretty sure. I guess it's just to avoid hardcoding :)